modules/ownadxBidAdapter.js

```diff
@@ -77,16 +78,16 @@ export const spec = {
       return bids;
     }
     const responseBid = {
+      width: response.width,
+      height: response.height,
       token: response.tokenId,
+      ttl: CREATIVE_TTL,
       requestId: response.slotBidId,
+      aType: response.adType || '1',
       cpm: response.cpm,
-      currency: response.currency || DEFAULT_CURRENCY,
-      adType: response.adType || '1',
-      width: response.adWidth,
-      height: response.adHeight,
-      ttl: CREATIVE_TTL,
       creativeId: response.creativeId || 0,
       netRevenue: response.netRevenue || false,
+      currency: response.currency || DEFAULT_CURRENCY,
```

Comment:
Hi, this appears to be a malicious attempt to silence the code duplication robot?